main.go

```diff
@@ -27,23 +27,28 @@ var (
 )
 
 // defaultCNIConf is a CNI configuration that enables network access to containers (docker-bridge style)
-var defaultCNIConf = fmt.Sprintf(`{
-	"cniVersion": "0.4.0",
-	"name": "%s",
-	"type": "bridge",
-	"bridge": "%s",
-	"isGateway": true,
-	"isDefaultGateway": true,
-	"promiscMode": true,
-	"ipMasq": true,
-	"ipam": {
-		"type": "host-local",
-		"ranges": [
-			[{
-				"subnet": "%s"
-			}]
-		]
-	}
+var defaultCNIConf = fmt.Sprintf(`
+{
+    "cniVersion": "0.4.0",
+    "name": "%s",
+    "plugins": [
+      {
+        "type": "bridge",
+        "bridge": "%s",
+        "isGateway": true,
+        "ipMasq": true,
+        "ipam": {
+            "type": "host-local",
+            "subnet": "%s",
+            "routes": [
+                { "dst": "0.0.0.0/0" }
+            ]
+        }
+      },
+      {
+        "type": "firewall"
```

Comment:
What does the firewall add?

Reply:
The plugin creates the forwarding rules to allow communication between containers. Like on faasd for gateway to be able to talk to nats and prom.

Reply:
Perhaps add a comment